src/components/divider/divider.component.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import StyledDivider from './divider.style';
+
+const Divider = ({
+  marginTop = 3,
+  marginBottom = 3,
+  marginLeft,
+  marginRight
+}) => {
+  return (
+    <StyledDivider
+      marginTop={ marginTop }
```

Comment:
could we add some `data-component='divider'` here? We will need it further in `cypress` tests